### PR TITLE
release notes rel.2.17

### PR DESCRIPTION
**Pull request by zhoujing2022**
**Description**

Changes proposed in this pull request:

- add file "2023-08-04-release-notes-2.17.md"
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: docs/kyma-release-notes/2023-08-04-release-notes-2.17.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- We have updated the [Monitoring stack]((https://github.com/kyma-project/kyma/pull/17877)):
+  - Prometheus to version 2.45.0 LTS
+  - Prometheus-operator to version 0.66.0
+
+### Removal of the Logging component
+The [Logging component](https://github.com/kyma-project/kyma/issues/15827), including the in-cluster Loki stack, has been removed, as announced in detail in this [blog post](https://blogs.sap.com/2023/06/02/removal-of-loki-based-application-logs-in-sap-btp-kyma-runtime/). Follow the [cleanup instructions](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.16-2.17.md) because updates to the component will no longer be shipped.
+
+## Telemetry
+The Telemetry stack has been upgraded:
+- [Otel-collector to version 0.81.0](https://github.com/kyma-project/kyma/pull/17807)
```
Comment (IwonaLanger) [inline docs/kyma-release-notes/2023-08-04-release-notes-2.17.md:17]:
```suggestion
- [OTel Collector to version 0.81.0](https://github.com/kyma-project/kyma/pull/17807)
```

File: docs/kyma-release-notes/2023-08-04-release-notes-2.17.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+We have implemented bug fixes for: 
+- A single TracePipeline referencing a non-existent Secret, resulting in a [crashing Trace Collector](https://github.com/kyma-project/telemetry-manager/issues/272).
+- A LogPipeline referencing a non-existent Secret, resulting in a [broken Fluent Bit configuration](https://github.com/kyma-project/telemetry-manager/issues/137).
+	
+## Service Mesh
+We have fixed a bug where the Istio sidecars tried to send spans to an unknown cluster-local address, causing [unneeded stress on CodeDNS](https://github.com/kyma-project/kyma/pull/17811).
+
+### Istio upgraded to 1.18.2
+In this release, we have upgraded Istio from 1.18.1 to 1.18.2. For more details on the changes, read the official [Istio 1.18.2 release notes](https://istio.io/latest/news/releases/1.18.x/announcing-1.18/upgrade-notes/).
```
Comment (IwonaLanger) [inline docs/kyma-release-notes/2023-08-04-release-notes-2.17.md:28]:
```suggestion
In this release, we have upgraded Istio from 1.18.1 to 1.18.2. For more details on the changes, read the official [Istio 1.18.2 release notes](https://istio.io/latest/news/releases/1.18.x/announcing-1.18.2/).
```

**Review by Disper — APPROVED** [review]

**Review by strekm — APPROVED** [review]
/meow

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**